### DISCUSSION TITLE
video + photo recording

### DISCUSSION THREAD

**Discussion by albiluc** [gopro/labs#372] [Feature Requests]
hi,
I'm new here but after searching I didn't find anything about that here...

for my use case it would be awesome to have (back) an old feature that was in hero 4session and other old models, 
which is the possibility to take images, every set amount of time, while recording video.

like when I'm recording sport events, start to end, and I also need some still images (without extrapolate frames from video of low quality) I have to set two cameras and many times I don't have space to fit both..

thanks to all
Alberto

**Comment by dnewman-gpsw** [comment]
Labs can't add video and photo features like this.  However frame extraction should not be low quality, 5k video and options increased bitrate would be more flexible than video + JPEGs.

**Comment by albiluc** [comment]
thanks for the explanation
